packages/mapper/src/mapper.ts

```diff
@@ -114,20 +106,14 @@ export default class Mapper {
 
     for (let i = 0; i < 20; i++) { // TODO: ensure that assigning cursor doesn't iterate
       if (!await this.currentCursor.hasNext()) {
-        this.messageBus.emit(
-          'mongosh:it',
-          { method: 'it', arguments: { result: 'no cursor' } }
-        );
+        this.messageBus.emit('mongosh:it');
```

Comment:
this is going to be emitted 20 times, is it ok?

Reply:
yea, i've been reading over my logs, and it's polluted with these. happy to remove!